### PR TITLE
DNM: rbd-replay* should ship in ceph-common

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/13172

### CODE REVIEW

**Comment by dachary** [comment]
DNM until #5790 is merged because it conflicts with it

File: ceph.spec.in [hunk 1]
```diff
@@ -659,6 +659,11 @@ fi
 %{_bindir}/ceph-crush-location
 %{_bindir}/rados
 %{_bindir}/rbd
+%{_bindir}/rbd-replay
+%{_bindir}/rbd-replay-many
+%if 0%{?fedora} || 0%{?rhel} == 6
+%{_bindir}/rbd-replay-prep
```
Comment (dachary) [inline ceph.spec.in]: There probably is a compilation conditional fix somewhere to not compile rbd-replay-prep on CentOS 7. Without it we get:
<pre>
Checking for unpackaged file(s): /usr/lib/rpm/check-files /tmp/release/CentOS/WORKDIR/BUILDROOT/ceph-0.87.1-37.gffb76d1.el7.x86_64^M
error: Installed (but unpackaged) file(s) found:^M
   /usr/share/man/man8/rbd-replay-prep.8.gz^M
^M
^M
RPM build errors:^M
    Installed (but unpackaged) file(s) found:^M
   /usr/share/man/man8/rbd-replay-prep.8.gz^M
</pre>

File: ceph.spec.in [hunk 2]
```diff
@@ -670,6 +675,9 @@ fi
 %{_mandir}/man8/ceph.8*
 %{_mandir}/man8/rados.8*
 %{_mandir}/man8/rbd.8*
+%{_mandir}/man8/rbd-replay.8*
+%{_mandir}/man8/rbd-replay-many.8*
+%{_mandir}/man8/rbd-replay-prep.8*
```
Comment (dachary) [inline ceph.spec.in]: Actually the problem is here it seems. It should not be included.

**Comment by dachary** [comment]
Please ignore my previous comments, I was just confused.

**Comment by smithfarm** [comment]
Moved the commit to #5790

**Pull request closed without merging**